docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
```

Comment:
What about adding a Windows section as TBD?  The best end state here is to actually have details for Windows, and to me it doesn't make sense for a page to change its title as we fill in more details.